middleware/common/include/casual/platform.h

```diff
@@ -168,6 +168,11 @@ namespace casual
             {
                constexpr size::type updates = 100;
```

Comment:
Is this still used now that you've switched to a timeout?

Reply:
No, you're right. 

I forgot to reference the issue, I'll remove the constant in the same force push.